app/helpers/defence_requests_helper.rb

```diff
@@ -87,4 +87,12 @@ def initial_date_type(initial_date, today, tomorrow)
     end
   end
 
+  def appropriate_adult_reason(defence_request)
+    if defence_request.appropriate_adult?
```

Comment:
I recall getting an error in test runs when this check was not present. This helper code is getting executed before the `boolean_with_explanation` helper code runs.